pkg/httpRequest/request.go

```diff
@@ -0,0 +1,20 @@
+package httpRequest
```

Comment:
package name should be not camel-case, should be lowercase only and short word like `httprequest` or `httpclient`

so in this case, I think should be `httpclient` it will be better.

ref: https://google.github.io/styleguide/go/decisions.html